tools/crm_mon.c

```diff
@@ -1598,6 +1591,10 @@ main(int argc, char **argv)
         return clean_up(CRM_EX_ERROR);
     }
 
+    add_output_args();
+
+    /* output_format MUST NOT BE CHANGED AFTER THIS POINT. */
```

Comment:
Don't the other parts of add_output_args() have to be done before creating the object?

Reply:
After looking at this again, it looks like it only matters for the the --xml-legacy option.  For all others, the order doesn't actually matter (well, as long as we don't call a function on the output object first).  I'll have to rearrange this.